### ISSUE TITLE
TipTac has been been blocked from an action only available to the Blizzard UI (while using abilities)

### ISSUE THREAD

**Issue by jarckn** [frozn/TipTac#169]
**Describe the bug**
Tried to use an ability, but got greeted by a "TipTac has been blocked" message.
This can be fixed by reloading the UI, but it will appear sometime again.

**To Reproduce**
I haven't been able to pin-point something down so far.

**Expected behavior**
I want to use my abilities.

**Screenshots**
![grafik](https://user-images.githubusercontent.com/1441097/212670852-07d35609-146e-47dd-8037-5038174df3bf.png)

**TipTac Reborn (please complete the following information):**
 - Version 22.12.15

**WoW (please complete the following information):**
 - Flavor: Retail
 - Version 10.0.2

**Additional context**
```
1x [ADDON_ACTION_FORBIDDEN] AddOn 'TipTac' hat versucht die geschützte Funktion 'UseAction()' aufzurufen.
[string "@!BugGrabber/BugGrabber.lua"]:480: in function <!BugGrabber/BugGrabber.lua:480>
[string "=[C]"]: in function `UseAction'
[string "@FrameXML/SecureTemplates.lua"]:364: in function `handler'
[string "@FrameXML/SecureTemplates.lua"]:690: in function <FrameXML/SecureTemplates.lua:672>
[string "@FrameXML/SecureTemplates.lua"]:704: in function <FrameXML/SecureTemplates.lua:697>
[string "@FrameXML/SecureTemplates.lua"]:746: in function `SecureActionButton_OnClick'
[string "@FrameXML/ActionButton.lua"]:109: in function `TryUseActionButton'
[string "@FrameXML/ActionButton.lua"]:144: in function `ActionButtonDown'
[string "ACTIONBUTTON11"]:2: in function <[string "ACTIONBUTTON11"]:1>
```

**Comment by jarckn** [comment]
I will try to get detailed info on how to reproduce this.

**Comment by akichusell** [comment]
is this issue a duplicate of https://github.com/frozn/TipTac/issues/149 ?

**Comment by jarckn** [comment]
> is this issue a duplicate of #149 ?

I don't think so. I checked this issue before posting this one but I couldn't replicate it the same way.
I will check on this in more detail in the evening.

**Comment by frozn** [comment]
Should be fixed with latest release [v23.01.23](https://github.com/frozn/TipTac/releases/tag/v23.01.23-release). Please try again with this version.
